### PR TITLE
Refactor/babel

### PR DESCRIPTION
**Pull request by GoPro16**
- Removed `url` prop from `AvLink` as it was changed to use `href`.
- Fixed `submitted` state on `AvForm` causing stories to crash and refactored to function component
- Simplified babel preset by just using single `babel-preset-react-app`.

### CODE REVIEW

File: packages/link/Link.js [hunk 1]
```diff
@@ -36,11 +36,10 @@ AvLink.defaultProps = {
 };
 
 AvLink.propTypes = {
-  url: PropTypes.string.isRequired,
   target: PropTypes.string,
   tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
   children: PropTypes.node,
-  href: PropTypes.string,
+  href: PropTypes.string.isRequired,
```
Comment (nylon22) [inline packages/link/Link.js:42]: This is a breaking change, no?

Comment (GoPro16) [reply packages/link/Link.js:42]: I just released the `href` change a few hours ago and needs to be set to required. Required prop types are just warnings and don't break the apps from what I can tell.

Comment (nylon22) [reply packages/link/Link.js:42]: People are currently using this component like:
```
<AvLink url="my-url-here"
```

I don't see anywhere in this change where `url` gets picked up from props and renamed to `href`.

**Comment by jordan-a-young** [comment]
My understanding was that the name of the prop changed which would be a
breaking change.

File: packages/link/Link.js (hunk 1 above)
Comment (GoPro16) [reply packages/link/Link.js:42]: https://github.com/Availity/availity-react/blob/master/packages/link/CHANGELOG.md

https://availity.github.io/availity-react/components/link/#href-string

Comment (nylon22) [reply packages/link/Link.js:42]: mea culpa. didn't see BREAKING CHANGE in commit message

**Review by nylon22 — APPROVED** [review]

**Pull request merged**